_includes/markdown/Systems.md

```diff
@@ -252,3 +253,94 @@ Once tuning moves beyond these main items, performance improvements will be mino
+
+Memcached and Redis are in memory data stores that are used for the WordPress object cache. Implementation of the object cache in WordPress code is covered extensively in the [PHP Performance](https://10up.github.io/Engineering-Best-Practices/php/#performance) section of the Best Practices and this section will focus on the hosting and setup of memcached and Redis.
+
+Memcached and Redis are used by WordPress as simple, in memory key-value stores. By being in memory and not having the possibility for complex queries, these tools provide blazing fast retrieval of data, usually in less than 1 millisecond. Common use cases of these data stores are:
+
+* Caching results of an external API call
+* Caching the results of a complex query
+* Storing full or partial pages 
+
+Items stored in these caching technologies are made up of 3 things: a key, a value, and an optional expiration time.  After the expiration time, items are evicted from the cache.  Items without an expiration time will persist in the cache until the cache is full, at which time the cache will evict the least recently used (LRU) item each time a new item needs to be stored. 
+
+### Sizing the Cache Pool
+
+The LRU eviction policy means that even a small cache can be effective and the most used data will always be in the cache. However, the most effective cache pool will never evict an item before the expiration time is reached and will have enough space for a healthy collection of items with no expiration time.  10up has tested various sizes of cache pools and has found 256 MB to be appropriate for most WordPress sites.  Complicated sites or large multisites may benefit from 512 MB and small sites on limited hardware can perform well with as little as 64 MB, but 256 MB should be used as a safe “rule of thumb”.  Above 256 MB, the cache hit rate and eviction rate usually do not improve no matter how large the cache pool is.
+
+A common misconception is that a full cache pool is a problem and should be avoided, or that a full cache means a bigger cache pool is needed. In reality, a full cache pool is the normal state of the cache with WordPress and means nothing.  No matter how big the cache pool, it will eventually become full.  This is because there will always be some cached items without an expiration time that will persist in the cache forever, until the cache is full and the LRU policy evicts it to make room for a new item.
+
+Even a very active cache will not stress a modern CPU, so a caching server can perform very well even with a single CPU core.  Similarly, since the cache activity is all in memory (unless persistence is enabled with Redis), a small and slow hard drive will work fine.  As long as network bandwidth is fast, latency is low, and enough RAM is available for the cache pool, nearly any server can host the cache. 
+
```

Comment:
Should we give some pointers on how to determine issues. ie. memcache stats phpmemcacheadmin/phpredisadmin and looking at ratios between set/get/evictions in context?

Reply:
maybe just link to tools section below

Reply:
I decided not to wade into that in this round as it is so different from site to site - there's not good numbers or direction I could come up with that was universally applicable.  A membership site will have a much different hit rate than a blog site.  A smaller cache pool will have a higher eviction rate than a larger pool, but the hit rate might be fine.  A large eviction rate doesn't mean much if you don't set expiration times, which is also fine.  I think I'm going to leave it as is - the tools section is fairly close to this one and I feel like all the sections written are important and connected.